.github/workflows/prepare-for-minor-release.yml

```diff
@@ -17,13 +17,16 @@ jobs:
 
   validate_source_branch:
     runs-on: ubuntu-latest
-    if: ${{ !contains(fromJSON('["main", "2.x"]'), github.ref_name) }}
     steps:
       - name: Validate source branch
         run: |
-          echo "Workflow triggered from invalid branch: ${{ github.ref_name }}" >> $GITHUB_STEP_SUMMARY
-          echo "Worfklow must be triggerd from a valid development branch (i.e. main or 2.x)" >> $GITHUB_STEP_SUMMARY
-          exit 1
+          dev_branches=("main" "2.x")
+          # Check if the search_string is present in the array
+          if ! [[ " ${dev_branches[@]} " =~ " ${GITHUB_REF_NAME} " ]]; then
+            echo "Workflow triggered from invalid branch: ${{ github.ref_name }}" >> $GITHUB_STEP_SUMMARY
+            echo "Worfklow must be triggered from a valid development branch (i.e. main or 2.x)" >> $GITHUB_STEP_SUMMARY
```

Comment:
```suggestion
            echo "Workflow must be triggered from a valid development branch (i.e. main or 2.x)" >> $GITHUB_STEP_SUMMARY
```

Reply:
Ha.  We noticed this last night but decided not to fix because we we're having to wait for workflows to run for every revision we made.  If we include that change, we'll need to apply that to 2.x as well 🙃